api-response.go

```diff
@@ -17,16 +17,198 @@
 package main
 
 import (
+	"encoding/xml"
 	"net/http"
 
 	"github.com/minio/minio/pkg/fs"
 )
 
-// Reply date format
 const (
+	// Reply date format
 	rfcFormat = "2006-01-02T15:04:05.000Z"
```

Comment:
Is this RFC3339 format? In that case,  variable should also mention the rfc name.